.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+
+jobs:
+  build:
+    working_directory: /home/reaction
+
+    docker:
+      - image: node:7
+
+    environment:
+      - METEOR_ALLOW_SUPERUSER: true
+
+    steps:
+      - setup_remote_docker
+      - checkout
+
+      # install OS dependencies
+      - restore_cache:
+          name: Restoring Meteor cache
+          key: meteor
+
+      - run: .circleci/install.sh
+
+      - save_cache:
+          name: Saving Meteor to cache
+          key: meteor
+          paths:
+            - ~/.meteor
+
+      # install app dependencies
+      - restore_cache:
+          name: Restoring node_modules cache
+          key: node_modules
+
+      - run: meteor npm install
+
+      - save_cache:
+          name: Saving node_modules to cache
+          key: node_modules
+          paths:
+            - /home/reaction/node_modules
+
+      # run tests
+      - restore_cache:
+          name: Restoring Meteor dev_bundle cache
+          key: dev_bundle
+
+      - run: reaction test
```

Comment:
Just a question, but does this new format only allow one line for the executable, as opposed to the old format where you could any number of executables?